lib/client-options.js

```diff
@@ -34,8 +35,10 @@ function defaultOptions () {
     },
     socketOptions: {
       connectTimeout: 5000,
+      defunctReadTimeoutThreshold: 64,
```

Comment:
I really like this capability.   What do you think about having some kind of 'heartbeat' timeout threshold?   In the java-driver if any heartbeat options message times out, [the connection is defuncted](https://github.com/datastax/java-driver/blob/2.1.6/driver-core/src/main/java/com/datastax/driver/core/Connection.java#L1094-L1102).  Part of me thinks this may not be necessary since the nodejs driver shares the control connection in the pool, so each connection should always stay pretty active and hit the current defunctReadTimeoutThreshold pretty quick.